src/main/java/io/luna/util/gui/PluginGuiController.java

```diff
@@ -1,6 +1,10 @@
 package io.luna.util.gui;
 
-import javafx.event.ActionEvent;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
```

Comment:
Import consistency, also avoid wildcards please.

Reply:
Again, Intellij's fault 😄 I'll disable automatic wildcards in my settings.